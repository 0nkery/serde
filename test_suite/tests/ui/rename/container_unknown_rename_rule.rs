#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(rename_all = "abc")]
struct S {
    name: u8,
    long_name: u8,
    very_long_name: u8,
}

fn main() {}
