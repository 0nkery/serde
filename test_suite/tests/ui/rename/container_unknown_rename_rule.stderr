error: unknown rename rule for #[serde(rename_all = "abc")]
 --> $DIR/container_unknown_rename_rule.rs:5:22
  |
5 | #[serde(rename_all = "abc")]
  |                      ^^^^^

error: aborting due to previous error

